src/views/ModelPlan/Table/index.tsx

```diff
@@ -181,7 +179,7 @@ const ModelPlansTable = ({
       },
       modelName: {
         id: 'modelName',
-        Header: homeT<string>('requestsTable.headers.name'),
```

Comment:
Can you fill me in on why you removed the type generic here?  They might be a little superfluous but were more descriptive